.rubocop.yml

```diff
@@ -3,6 +3,8 @@ AllCops:
     - mixlib-install.gemspec
 Style/AlignParameters:
   EnforcedStyle: with_first_parameter
+Style/BlockDelimiters:
```

Comment:
If I remove the .rubocop.yml I get oodles of `spec/mixlib/install/product_spec.rb:70:19: C: Style/BlockDelimiters: Avoid using {...} for multi-line blocks.
    let(:product) {` type things.... any suggestions? I can modify the code in all those places but that seems... wrong?